### PR TITLE
Add new CV32E40Pv2 Project Concept proposal

### PR DESCRIPTION
**Pull request by pascalgouedo**

### CODE REVIEW

File: program/Project Descriptions and Plans/CV32E40Pv2/Project_Concept_for_CV32E40Pv2_June 28.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+# OpenHW Project Concept Proposal: CV32E40Pv2
+*The PC proposal introduces the project and explains the market drivers and the "why"* 
+
+# Title of Project - "CV32E40Pv2"
+# Project Concept Proposal
+## Date of proposal - 2021-06-28
+## Author(s) - Olivier Montfort (Dolphin Design), Pascal Gouedo (Dolphin Design), Yoann Pruvost (Dolphin Design)
+
+## High Level Summary of project, project components, and deliverables
+This proposal is about verifying PULP basic instructions disabled during CV32E40P v1 phase improving performances, power consumption and even code size depending of application programs.
```
Comment (davideschiavone) [inline program/Project Descriptions and Plans/CV32E40Pv2/Project_Concept_for_CV32E40Pv2_June 28.md:10]: This proposal is about verifying XPULP, RVF, and Zfinx ISA extensions that were disabled [..]

File: program/Project Descriptions and Plans/CV32E40Pv2/Project_Concept_for_CV32E40Pv2_June 28.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  Verify RISC-V standard F extension instructions (Zfinx and non-Zfinx) with coupled Core-FPnew?
+* or Merge ETH CV-X-IF and only Verify this interface?
+
+### Potential future design enhancements
+* Conversion of PULP Bit Manipulation & SIMD instruction to ratified RISC-V B and P extensions.
+* Add Zce extension for further code-size reduction (not required for Multicore cluster).
+* Add bus attributes and errors.
+
+## Who would make use of OpenHW output
+Companies needing more performances, less power consumption or smaller code size for standard control or DSP-like computing applications.
```
Comment (davideschiavone) [inline program/Project Descriptions and Plans/CV32E40Pv2/Project_Concept_for_CV32E40Pv2_June 28.md:47]: less energy consumption, not power

**Review by davideschiavone — CHANGES_REQUESTED** [review]

**Review by davideschiavone — APPROVED** [review]

File: program/Project Descriptions and Plans/CV32E40Pv2/Project_Concept_for_CV32E40Pv2_June 28.md (hunk 1 above)
Comment (pascalgouedo) [reply program/Project Descriptions and Plans/CV32E40Pv2/Project_Concept_for_CV32E40Pv2_June 28.md]: updated

File: program/Project Descriptions and Plans/CV32E40Pv2/Project_Concept_for_CV32E40Pv2_June 28.md (hunk 2 above)
Comment (pascalgouedo) [reply program/Project Descriptions and Plans/CV32E40Pv2/Project_Concept_for_CV32E40Pv2_June 28.md]: updated

**Comment by davideschiavone** [comment]
Hi @pascalgouedo , you need to sign up the Eclipse ECA with your GitHub account before merging

**Comment by pascalgouedo** [comment]
Filing Eclipse ECA on-going

**Pull request merged**